README.md

```diff
@@ -234,20 +234,24 @@ if support for their modification has been implemented.
 ```yaml
 firewall:
   - firewalld_conf:
-      allow_zone_drifting: "no"
+      allow_zone_drifting: no
```

Comment:
```suggestion
      allow_zone_drifting: false
```
https://github.com/redhat-cop/automation-good-practices/tree/main/coding_style#yaml-and-jinja2-syntax "Use true and false for boolean values in playbooks"